app/lib/konnectors.js

```diff
@@ -49,8 +49,8 @@ function waitForKonnectorReady (cozy, konnector, timeout) {
 
     const idInterval = setInterval(() => {
       cozy.data.find(KONNECTORS_DOCTYPE, konnector._id)
-        .then(konnector => {
-          if (konnector.state === STATE_READY) {
+        .then(konnectorResult => {
```

Comment:
Yes, this causes the issue that the permission document linked to the konnector (which comes from the POST to the /konnectors route is lost. But I agree there may be cleaner solutions.